content/en/synthetics/mobile_app_testing/mobile_app_tests/restricted_networks.md

```diff
@@ -41,6 +41,35 @@ The following is the list of IP ranges associated with the real devices used for
 `185.94.24.0/22`</br>
 `34.96.70.78`</br>
 
+## Troubleshooting
+
+If you experience issues with Mobile App Testing on restricted networks, use the following troubleshooting guidelines. If you need further assistance, contact [Datadog support][1].
+
+### Unable to launch recorder
+
+To launch the Mobile Application Testing (MAT) Recorder, Datadog needs to establish UDP/TCP TURN connections to enable WebRTC connections. If the user is behind a restrictive network (such as a strict firewall or VPN), these connections may fail, leading to the following error:
```

Comment:
```suggestion
To launch the Mobile Application Testing (MAT) Recorder, Datadog needs to establish UDP/TCP TURN connections to enable WebRTC connections. If the user is behind a restrictive network (such as a strict firewall or VPN), these connections may fail, leading to a "Device unexpectedly disconnected" error:
```
In order to support search, I don't recommend using an image alone to describe the error. Most people search for the text of the error they received.